modules/apps/frontend-js/frontend-js-web/src/main/resources/META-INF/resources/liferay/global.es.js

```diff
@@ -55,8 +60,16 @@ Liferay.Address = {
 
 Liferay.SideNavigation = SideNavigation;
 
+Liferay.Store = {
```

Comment:
1. Should be `Liferay.Util.Store`
1. Moar tests! (We would notice this immediately if we had any test)